Makefile

```diff
@@ -382,15 +382,28 @@ tests/%.prove: tests/% $(prove_kompiled)
 # Dexter proofs
 
 dexter_spec_modules = DEXTER-SPEC                          \
-                      DEXTER-ADDLIQUIDITY-POSITIVE-SPEC    \
                       DEXTER-ADDLIQUIDITY-NEGATIVE-SPEC    \
-                      DEXTER-REMOVELIQUIDITY-POSITIVE-SPEC \
+                      DEXTER-ADDLIQUIDITY-POSITIVE-SPEC    \
+                      DEXTER-DEFAULT-SPEC                  \
                       DEXTER-REMOVELIQUIDITY-NEGATIVE-SPEC \
-                      DEXTER-SETMANAGER-SPEC               \
+                      DEXTER-REMOVELIQUIDITY-POSITIVE-SPEC \
                       DEXTER-SETBAKER-SPEC                 \
                       DEXTER-SETLQTADDRESS-SPEC            \
+                      DEXTER-SETMANAGER-SPEC               \
+                      DEXTER-TOKENTOXTZ-FA12-NEG-1-SPEC    \
+                      DEXTER-TOKENTOXTZ-FA12-NEG-2-SPEC    \
+                      DEXTER-TOKENTOXTZ-FA12-NEG-3-SPEC    \
+                      DEXTER-TOKENTOXTZ-FA12-NEG-4-SPEC    \
+                      DEXTER-TOKENTOXTZ-FA12-SPEC          \
+                      DEXTER-TOKENTOXTZ-FA2-NEG-1-SPEC     \
+                      DEXTER-TOKENTOXTZ-FA2-NEG-2-SPEC     \
+                      DEXTER-TOKENTOXTZ-FA2-NEG-3-SPEC     \
+                      DEXTER-TOKENTOXTZ-FA2-SPEC           \
                       DEXTER-UPDATETOKENPOOL-SPEC          \
-                      DEXTER-DEFAULT-SPEC
+                      DEXTER-XTZTOTOKEN-FA12-NEG-SPEC      \
+                      DEXTER-XTZTOTOKEN-FA12-POS-SPEC      \
+                      DEXTER-XTZTOTOKEN-FA2-NEG-SPEC       \
+                      DEXTER-XTZTOTOKEN-FA2-POS-SPEC       \
```

Comment:
The naming is confusing me. Is the `TOKENTXTZ-FA2-SPEC` a positive spec, or a collection of specs, for example.

Also, I'd prefer if we stick to one naming convention: either write out `NEGATIVE` and `POSITIVE` everywhere, or shorten it to `NEG` and `POS` everywhere. I would prefer the former.